File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    const R = wallpaper.length;
    const C = wallpaper[0].length;
    const files_row = [];
    const files_col = [];
    let fileCount = 0;
    
    for (let r = 0; r < R; r++) {
        for (let c = 0; c < C; c++) {
            if (wallpaper[r][c] === '.') continue;

            fileCount++;
            files_row.push(r);
            files_col.push(c);
        }
    }
    
    files_col.sort((a, b) => a - b);
    
    return [
            files_row[0],
            files_col[0],
            files_row[fileCount - 1] + 1,
            files_col[fileCount - 1] + 1
           ];
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    lux = 50
    luy = 50
    rdx = 0
    rdy = 0
    
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == '#':
                lux = min(lux, i)
                luy = min(luy, j)
                rdx = max(rdx, i)
                rdy = max(rdy, j)
                                
    return [lux, luy, rdx+1, rdy+1]